bcda/api/alr.go

```diff
@@ -31,17 +30,13 @@ func (h *Handler) alrRequest(w http.ResponseWriter, r *http.Request, reqType ser
 		panic("Request parameters must be set prior to calling this handler.")
 	}
 
-	if err := h.validateRequest(rp.ResourceTypes, ad.CMSID); err != nil {
-		oo := responseutils.CreateOpOutcome(fhircodes.IssueSeverityCode_ERROR, fhircodes.IssueTypeCode_EXCEPTION, responseutils.RequestErr,
-			err.Error())
-		responseutils.WriteError(oo, w, http.StatusBadRequest)
-		return
-	}
-
-	req := service.DefaultAlrRequest
-	if reqType == service.Runout {
-		req = service.RunoutAlrRequest
-	}
+	// Currently, we don't do anything with resource types, thus commented out
```

Comment:
There is no need for this code now, since we do not except resource type param for ALR. In the future we may, so commenting this out for now.